spec/views/case_workers/claims/show_spec.rb

```diff
@@ -4,11 +4,22 @@
   subject { rendered }
 
   let(:case_worker) { create(:case_worker) }
+  let(:claim) { create(:claim) }
 
   before do
     initialize_view_helpers(view)
     sign_in(case_worker.user, scope: :user)
     allow(view).to receive(:current_user_persona_is?).and_return(false)
+    params[:tab] = 'information'
+
+    # Set up the controller instance variables
```

Comment:
More nitpicking - I probably wouldn't bother with the comment here, I think it's clear what the subsequent lines are doing so it doesn't add much.